src/serac/physics/base_physics.hpp

```diff
@@ -115,7 +115,7 @@ class BasePhysics {
    * @param parameter_name The name of the parameter to generate
    *
    * @note The user is responsible for managing the lifetime of this object. It is required
-   * to exist whenever advanceTimestep, solveAdjoint, or computeSensitivity is called.
+   * to exist whenever advanceTimestep, reverseAdjointTimestep, or computeTimestepSensitivity is called.
```

Comment:
Minor comment, but does "reverse adjoint timestep" have sort of a double negative meaning, in the sense that the adjoint approach is already moving backwards through time? What do you think about a more "symmetric" construction with `advanceTimestep` such as `reverseTimestep` or `retreatTimestep`? Or even `advanceAdjointTimestep`, although that might cause confusion for someone using Serac as a (primal) solver library.

Reply:
Keeping name for now. Can discuss in a bug fix day.